### PR TITLE
Add basic information to xrframe.md.

### PR DESCRIPTION
**Pull request by jpmedley**
Sans examples.

### CODE REVIEW

File: webxr-device-api/xrframe.md [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
-
-An instance of this object is passed to each call of the callback provided to `XRSession.<a href="requestanimationframe">requestAnimationFrame()<a>`.
+The **`XRFrame`** interface of the WebXR Device API represents a snapshot of the state of all tracked objects for an XRSession. Tracked objects include the headset, controllers, and possibly hands, if they're being exposed as input sources. An instance of this object is passed to each call of the callback provided to `XRSession.<a href="requestanimationframe">requestAnimationFrame()<a>`.
 
 ## Properties
 
 <dl>
   <dt>session \{\{readonlyinline\}\}</dt>
-  <dd>TBD</dd>
+  <dd>A reference to the current <code><a href="xrsession">XRSession</a></code>.</dd>
```
Comment (toji) [inline webxr-device-api/xrframe.md]: The use of the term "current" here may be confusing. I'd suggest something along the lines of "A reference to the `XRSession` that produced the frame"?

File: webxr-device-api/xrframe.md [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
 
 <dl>
   <dt>getPose()</dt>
-  <dd>TBD</dd>
-  <dt>getViewerPose()</dt>
-  <dd>TBD</dd>
+  <dd>Returns an <code><a href="xrpose">XRPose</a></code> instance which describes a position and orientation in space relative to an <code><a href="xrspace">XRSpace</a></code>.</dd>
+
+  <dt>getViewerPose(xrReferenceSpace)</dt>
+  <dd>Returns an <code><a href="xrpose">XRViewerPose</a></code> relative to the supplied <code><a href="xrreferencespace">XRReferenceSpace</a></code> describing the state of a viewer of the XR scene. The viewer may represent a tracked piece of hardware, the observed position of the user's head relative to the hardware, or </code></dd>
```
Comment (toji) [inline webxr-device-api/xrframe.md]: > The viewer may represent a tracked piece of hardware, the observed position of the user's head relative to the hardware, or 

or what? You've got me on the edge of my seat in anticipation. 😉

Comment (jpmedley) [reply webxr-device-api/xrframe.md]: I meant to send you a question about this. The next line in the spec is “some other means of computing a series of viewpoints into the XR scene.” Would an API user ever need to know this or is it only relevant to API implementors?

Comment (toji) [reply webxr-device-api/xrframe.md]: It's only really relevant to implementors. The important bit is that developers don't immediately assume the viewer is aligned either with the users head (may not be in the case of something like handheld AR) or a particular piece of tracked hardware.

File: webxr-device-api/xrframe.md (hunk 1 above)
Comment (jpmedley) [reply webxr-device-api/xrframe.md]: Done.

File: webxr-device-api/xrframe.md (hunk 2 above)
Comment (jpmedley) [reply webxr-device-api/xrframe.md]: Fixed.

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**